patches/api/0329-Expose-Player-Idle-Time.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: thamid-gamer <60953955+thamid-gamer@users.noreply.github.com>
+Date: Sat, 31 Jul 2021 03:42:06 -0400
+Subject: [PATCH] Expose Player Idle Time
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 3ec1be36b90dbedb8631135555da4b69110e4791..72a25d33defac32498738afc5789cbb98caf2477 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -979,7 +979,7 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+      * @param fadeInTicks  ticks to fade-in
+      * @param stayTicks    ticks to stay visible
+      * @param fadeOutTicks ticks to fade-out
+-     * @deprecated Use {@link #showTitle(net.kyori.adventure.title.Title)} 
++     * @deprecated Use {@link #showTitle(net.kyori.adventure.title.Title)}
```

Comment:
```suggestion
+     * @deprecated Use {@link #showTitle(net.kyori.adventure.title.Title)} 
```
Can you revert this back to the way it was? You probably shouldn't be touching this with this patch.

Reply:
how do I stop intellij from doing stuff like this lol

Reply:
Are you using IntelIJ's indentation function? That might also change stuff like trailing  spaces.